source/supported-hardware/cml-rvp.rst

```diff
@@ -0,0 +1,125 @@
+.. _comet-lake-rvp:
+
+Comet Lake CRB Boards
+-----------------------
+
+.. note:: 10th Generation Intel® Core Processor is formally known as |Comet Lake S|.
+
+Supported Boards
+^^^^^^^^^^^^^^^^^^^^^
+
+|SPN| supports **CML S RVP with PCH H, CML S RVP with PCH V, CML S RVP with H420E** platform.
+
+
+Building
+^^^^^^^^^^
+
+To build |SPN| for |CML| platform::
+
+    python BuildLoader.py build cml
+
+Replace ``cml`` with ``cmlv`` for PCH V.
+
+The output images are generated under ``Outputs`` directory.
+
+
+Stitching
+^^^^^^^^^^
+
+1. Gather |CML| IFWI firmware image
+
+  Users can either download the full IFWI image if the IFWI image release is available, or read the exising IFWI image on the board using SPI programmer.
+  This image contains additional firmware ingredients that are required to boot on |CML|.
+
+.. note::
+  ``StitchLoader.py`` currently only supports stitching with boot guard feature **disabled**.

```

Comment:
Adding 
To stitch boot guard enabled IFWI, please use StitchIfwi.py instead.

Reply:
I've added this.